JSQMessagesViewController/Views/JSQMessagesCollectionView.m

```diff
@@ -37,6 +37,29 @@ - (void)jsq_configureCollectionView;
 
 @implementation JSQMessagesCollectionView
 
+- (void)setContentOffset:(CGPoint)contentOffset {
+    if (self.contentSize.height < 1 &&
+        CGPointEqualToPoint(CGPointZero, contentOffset)) {
+        // [UIScrollView _adjustContentOffsetIfNecessary] resets the content
+        // offset to zero under a number of undocumented conditions.  We don't
+        // want this behavior; we want fine-grained control over the default
+        // scroll state of the message view.
+        //
+        // [UIScrollView _adjustContentOffsetIfNecessary] is called in
+        // response to many different events; trying to prevent them all is
+        // whack-a-mole.
+        //
+        // It's not safe to override [UIScrollView _adjustContentOffsetIfNecessary],
+        // since its a private API.
+        //
+        // We can avoid the issue by simply ignoring attempt to reset the content
+        // offset to zero before the collection view has determined its content size.
+        return;
+    }
```

Comment:
The new fix.

Reply:
This **seems** reasonable enough. 

The goal here is staying scrolled to bottom, right?